.github/workflows/security-full-suite.yml

```
@@ -0,0 +1,392 @@
+          if-no-files-found: ignore
+          retention-days: 30
+
+      - name: Evaluate SAST result
+        if: always()
+        run: |
+          if [ ! -f semgrep-results.json ]; then
+            echo "No Semgrep results file -- scan may have failed"
+            exit 1
+          fi
+          COUNT=$(python3 -c "
+import json
```

actionlint (1.7.11)

[error] could not parse as YAML: could not find expected ':'

(syntax-check)